helm/charts/td-agent-barito/templates/tdagentconfig.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: td-agent-config
+  namespace: {{ .Release.Namespace }}
+
+#################################################################################
```

Comment:
I don't think this is the correct mechanism, the commented out codes won't be reflected in user configMap. Also source should not be commented, because it will mostly be the same.

Rather you should create a readme for explaining to user how they can modify the config map for adding matches.